testutil/sims/address_helpers.go

```diff
@@ -35,7 +35,7 @@ func AddTestAddrsFromPubKeys(bankKeeper BankKeeper, stakingKeeper StakingKeeper,
 
 // AddTestAddrs constructs and returns accNum amount of accounts with an
 // initial balance of accAmt in random order
-func AddTestAddrs(bankKeeper BankKeeper, stakingKeeper StakingKeeper, ctx sdk.Context, accNum int, accAmt math.Int) []sdk.AccAddress {
+func AddTestAddrs(bankKeeper BankKeeper, stakingKeeper StakingKeeper, ctx context.Context, accNum int, accAmt math.Int) []sdk.AccAddress {
```

Comment:
Can you backport this file manually in 0.52?

Reply:
yeah i can open a PR to v0.52.x

Reply:
Thanks, let's backport that whole file I think, as more changes happen in previous PRs I see (https://github.com/cosmos/cosmos-sdk/pull/22754/files#diff-930b2d983970f416204d5439b3ddd15e9b8f32b0c080b7fa8d84f34245d43191)

Reply:
Oh and this too: https://github.com/cosmos/cosmos-sdk/pull/22667#pullrequestreview-2501661104 :D